buildlib/pr/main.yml

```diff
@@ -1,15 +1,16 @@
 variables:
   DOCKER_OPT_VOLUMES: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
-  DOCKER_OPT_GPU: --gpus all --ulimit memlock=-1:-1 --device=/dev/infiniband/
+  DOCKER_OPT_IB: --ulimit memlock=-1:-1
+  DOCKER_OPT_IB_GPU: --gpus all --ulimit memlock=-1:-1 --device=/dev/infiniband/
 
 resources:
   containers:
     - container: centos7
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7:4
-      options: $(DOCKER_OPT_VOLUMES)
+      options: $(DOCKER_OPT_VOLUMES) $(DOCKER_OPT_IB)
```

Comment:
why needed here?

Reply:
can you point or copy/paste the failure?
i would not expect this PR to expose new failures in dockers, but only report a better error message in case of already existing failing cases.

Reply:
```
2022-03-01T08:52:58.1401374Z Got 0 errors and 4 warnings during the test
2022-03-01T08:52:58.1403598Z [     INFO ] < /__w/1/s/contrib/../src/uct/ib/base/ib_md.c:1193 RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited' >
2022-03-01T08:52:58.1405949Z [     INFO ] < /__w/1/s/contrib/../src/uct/ib/base/ib_md.c:1193 RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited' >
2022-03-01T08:52:58.1408388Z [     INFO ] < /__w/1/s/contrib/../src/uct/ib/base/ib_md.c:1193 RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited' >
2022-03-01T08:52:58.1410715Z [     INFO ] < /__w/1/s/contrib/../src/uct/ib/base/ib_md.c:1193 RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited' >
2022-03-01T08:52:58.1412340Z [  FAILED  ] tcp/test_ucp_sockaddr_protocols_err.tag_rndv_unexp/6, where GetParam() = tcp,cuda_copy,rocm_copy/recv_stop (232 ms)
2022-03-01T08:52:58.1413623Z [ RUN      ] tcp/test_ucp_sockaddr_protocols_err.tag_eager_32_unexp/2 <tcp,cuda_copy,rocm_copy/send_stop,sa_data_v2>
2022-03-01T08:52:58.1416061Z [1646124777.481760] [9a224e4fc163:27518:0]           ib_md.c:1193 UCX  WARN  RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited'
2022-03-01T08:52:58.1418624Z [1646124777.482137] [9a224e4fc163:27518:0]           ib_md.c:1193 UCX  WARN  RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited'
2022-03-01T08:52:58.1421080Z [1646124777.484896] [9a224e4fc163:27518:0]           ib_md.c:1193 UCX  WARN  RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited'
2022-03-01T08:52:58.1423748Z [1646124777.485235] [9a224e4fc163:27518:0]           ib_md.c:1193 UCX  WARN  RDMA transports are disabled because max locked memory limit (64 kbytes) is too low. Please set max locked memory (ulimit -l) to 'unlimited'
2022-03-01T08:52:58.1425206Z [     INFO ] server listening on 172.17.0.4:46574
2022-03-01T08:52:58.1426115Z [     INFO ] ignoring error Endpoint timeout on endpoint 0x7fd0e2c85000
2022-03-01T08:52:58.1427134Z [     INFO ] < operation returned error: Endpoint timeout >
2022-03-01T08:52:58.1428053Z /__w/1/s/contrib/../test/gtest/common/test.cc:344: Failure
2022-03-01T08:52:58.1428767Z Failed
```

That is the log from the last CI. It seems like the memory limit does not cause any errors but the test fails due to timeout that can be caused by disabling the RDMA transport.

Reply:
pls see https://github.com/openucx/ucx/pull/7930#discussion_r816613271